src/definition/scheduler/IJob.ts

```diff
@@ -0,0 +1,18 @@
+import { IHttp, IModify, IPersistence, IRead } from './../accessors';
+import { IJobResult } from './IJobResult';
+import { Schedule } from './Schedule';
+
+export interface IJob {
```

Comment:
Maybe you would want to store which user scheduled the job so admins can see who did what.
Or do you think that that's more an optional thing to store in `data`?

Reply:
Yes sir. As you wouldn't want to limit the system to requiring a user since an App could provide "daily check ins" and that would need to be automatically scheduled when the app is installed without requiring a user to schedule it.

Reply:
Added a `creator: IUser` property to the job and removed the `data` property since Apps should be using the persistence layer to store data.